easyfinance.client/src/app/core/services/user.service.ts

```diff
@@ -55,4 +58,20 @@ export class UserService {
       return this.refreshUserInfo();
     }));
   }
+
+  public deleteUser(token?: string) {
+    const options = token
+      ? {
+          body: { ConfirmationToken: token }, 
+        }
+      : undefined; 
```

Comment:
```suggestion
  public deleteUser() {
    const options = this.tokenService..getToken()
      ? {
          body: { ConfirmationToken: this.tokenService..getToken() }, 
        }
      : undefined; 
```

I suggest get the token from the tokenService to avoid generate another token with existing token.

Reply:
As I said, It's just a suggestion. If you don't liked you don't need to apply. just closed it.

Reply:
Okay !